packages/sui-bundler/README.md

```diff
@@ -54,6 +56,29 @@ $ sui-bundler dev
 
 While developing your app, you will have HMR (Hot Module Reloading). Default port for your website is 3000, but it will assign automatically a free port for you if this one is busy.
 
+#### How to link packages
+
+`sui-bundler` give us the oportunity to link `npm` packages with `--link-package`. This argument accepts relative and absolute paths.
+The `npm link` is cancelled as soon as we stop/cancel the terminal process.
+
+**Requeriments**
```

Comment:
typo

Reply:
lol. I've always written it that way. I saw the light.